templates/incl-editor-and-output.html

```diff
@@ -85,9 +85,19 @@
       </div>
     </div>
     <div class="flex flex-col order-3" id="code_output" style="height: 22rem;">
-      <!-- tabindex=0 makes the div focusable -->
-      <div id="output" tabindex=0 class="flex-1 rounded p-2 px-3 bg-gray-900 color-white w-full text-lg overflow-auto" style="min-height: 3rem"></div>
-      <div id="turtlecanvas" class="flex-0 ltr:pl-4 rtl:pr-4"></div>
+  <!-- tabindex=0 makes the div focusable -->
+        <div id="output" tabindex=0 class="flex-1 rounded p-2 px-3 bg-gray-900 color-white w-full text-lg" style="position:relative"
+             style="min-height: 3rem">
+             <div class="dropdown" style="height: 30px; width: 62px;margin-left: 88%; margin-top: -54px;">
```

Comment:
It would be nice if we solved this with a bit more "css friendly" styling. For example with something like this:

`ltr:right-0 rtl:left-0 absolute z-10 mx-2 mt-2`

Instead of styling the button we should be styling the dropdown-menu itself, containing the button.

Reply:
After some puzzling, I've replaced most of the CSS inline code with tailwind classes :) I hope its better now